ui/analyse/src/ctrl.ts

```diff
@@ -128,6 +128,11 @@ export default class AnalyseCtrl {
     makeStudy?: typeof makeStudyCtrl,
   ) {
     this.data = opts.data;
+    if (opts.data.game.variant.key === 'standard' || opts.data.game.variant.key === 'fromPosition') {
+      const new_fen = this.returnStandardCastlingFen(opts.data.game.fen);
```

Comment:
please mind the project naming conventions. `new_fen` => `newFen`. More occurences in the pull request.